packages/docs/modules/ROOT/pages/public-deploy.adoc

```diff
@@ -100,6 +100,14 @@ module.exports = {
       provider: () => new HDWalletProvider(process.env.DEV_MNEMONIC, "https://ropsten.infura.io/v3/" + infuraProjectId),
       networkId: 3,       // Ropsten's id
     },
+    rinkeby: {
+      provider: () => new HDWalletProvider(process.env.DEV_MNEMONIC, "https://rinkeby.infura.io/v3/" + infuraProjectId),
```

Comment:
we could use template literals here :nerd_face:, they are easier to read!

Reply:
I’ll have to do another PR